### ISSUE TITLE
[Bug] - `Radio Group` not focusable in initial (deselected) state, some other ARIA issues

### ISSUE THREAD

**Issue by o-hook** [porsche-design-system/porsche-design-system#4062]
Description
---

**Keyboard issue:**
The `Radio Group` component is not keyboard accessible (focusable) if none of the items has a `checked` state. If no item is selected, then the first one needs to be focusable.

**SEMANTICS/ARIA issues:**
- A radio group must be wrapped inside a `fieldset` element with a `legend` as the overarching label (and not with a `label` element). The `aria-labelledby` attribute has a wrong value (and can probably removed if a `fieldset` and `legend` is used.
- If the `Radio Group` is in `error` state, the error message is not announced if a radio button has focus. This is because the `aria-labelledby` attribute has no reference to the ID of the error message (cross shadowDOM issue).
- `aria-describedby` attribute of the radio-group is missing. This should point to the ID of the overarching description.
- Screenreader announcement of number of options not correct: "Option one of one" (instead of sum of all options)
<img width="856" height="302" alt="Image" src="https://github.com/user-attachments/assets/74a91e65-4132-42a4-9c20-525e65021987" />

Notes  
---
See also official documentation: https://developer.mozilla.org/en-US/docs/Web/HTML/Reference/Elements/input/radio

Acceptance criteria
---
- [ ] `Radio Group` is fully keyboard accessible
- [ ] Overarching label of `Radio Group` is semantically correct

Subtasks
---
- [ ] Task
